pkgs/by-name/fr/freej2me/package.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, ant
+, jdk
+, stripJavaArchivesHook
+, SDL2
+, freeimage
+, writeShellScriptBin
+}:
+
+stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation {
```
`finalAttrs` isn't used anywhere